_posts/2024-02-19-jco-1.0.md

```diff
@@ -0,0 +1,290 @@
+---
+title: "Announcing Jco 1.0"
+author: "Yosh Wuyts"
+date: "2024-02-21"
+github_name: "yoshuawuyts"
+excerpt_separator: <!--end_excerpt-->
+---
+
+We’re happy to announce the 1.0 release of [jco]: a native Javascript
+WebAssembly runtime and toolchain built for WebAssembly Components. Jco can
+natively run Wasm Components inside of Node.js, making it easier to take
+libraries written in different programming languages and execute them using the
+Node.js runtime. And by implementing [WASI
+0.2](https://bytecodealliance.org/articles/WASI-0.2), those components can
+access the network, filesystem, and other system APIs available in the Node.js
+runtime.
+
+<!--end_excerpt-->
+
+Our goal is for jco to be a comprehensive tool for all Wasm-related operations
+for JavaScript. With Jco 1.0 we're stabilizing the Node.js Wasm Component
+runtime, as well as the toolchain to take Wasm Components written in other
+languages and import them from JavaScript. What is also included but not yet
+stable are: the browser runtime for Wasm Components and the JavaScript to Wasm
+Component compiler. This functionality can already be used experimentally, but
+needs additional work to bring it up to the standard we expect.
+
+## JavaScript Toolchains in the Bytecode Alliance
+
+Jco is the third JS toolchain project which is part of the Bytecode Alliance. To
+help people navigate this, here is a quick overview of the three projects and
+what their purpose is at the time of writing:
+
+- [Javy](https://github.com/bytecodealliance/javy): A JavaScript → WASI
+toolchain built for WASI 0.1. This work predates Wasm Components, but Component
+support may be added in the future.
+- [ComponentizeJS](https://github.com/bytecodealliance/ComponentizeJS): A
+JavaScript → WASI toolchain built for WASI 0.2 and Wasm Components. This project
+is newer and not yet considered stable.
+- [Jco](https://github.com/bytecodealliance/jco): A comprehensive WASI compiler
+toolchain and runtime for JavaScript. This includes a WASI → JS toolchain built
+for WASI 0.2 and Wasm Components.
+
+The goal is for Jco to eventually evolve into a comprehensive toolchain for all
+WASI and JavaScript needs. The command `jco componentize` uses ComponentizeJS
```

Comment:
`WASI support and JavaScript component needs`?

Reply:
done!